docs/technologies.md

```diff
@@ -0,0 +1,80 @@
+# Technologies
+
+This document provides a brief overview of some of the key technologies we use in the storage service.
+
+## AWS services
+
+<table>
+  <tr>
+    <td>
+      <img src="./images/Amazon-Simple-Storage-Service_64@5x.png">
+    </td>
+    <td>
+      <strong>S3</strong> – petabyte-scale object storage.
+      We use it for permanent storage of assets, temporary/working storage, and to hold storage "manifests" (our JSON representation of a bag).
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <img src="./images/Arch_Amazon-DynamoDB_64@5x.png">
+    </td>
+    <td>
+      <strong>DynamoDB</strong> – a NoSQL database that we use as a key-value store.
+      We use it to track ingests, store information about versions, and lock around certain processes.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <img src="./images/Arch_Amazon-Simple-Queue-Service_64@5x.png">
+      <img src="./images/Arch_Amazon-Simple-Notification-Service_64@5x.png">
+    </td>
+    <td>
+      <strong>SNS/SQS</strong> – inter-app message queues.
+      Our apps form a pipeline: an app gets a message from an SQS queue, does some work, then sends an SNS notification to the next app.<br/><br/>
+      We use SNS <em>topics</em> and SQS <em>queues</em>.
+      Queues can subscribe to the output of topics, in a many-to-many relationship.
+      This allows one app to send messages to multiple apps, or one app to receive messages from multiple apps.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <img src="./images/Arch_AWS-Fargate_64@5x.png">
+    </td>
+    <td>
+      <strong>ECS/Fargate</strong> – a serverless container runtime.
+      We package our services in Docker images, and then Fargate actually runs the containers, without us having to provision VMs/servers to run them on.
+      Fargate is a subservice of Elastic Container Service, or ECS.
```

Comment:
Worth mentioning cost savings?

Reply:
What were you thinking? The Fargate cost story is sufficiently complicated that I'm not sure what I could write in a sentence or two.